### PR TITLE
Create Main Section of my Portfolio (mobile version)

### PR DESCRIPTION
**Pull request by Jord4N-AC**
In the first milestone of creating my portfolio website, I have set up a new repository and prepared it for development using best practices, build the first 2 sections of the mobile website, meeting the following:

### Requirements
- Build the toolbar (or header)
- Build headline section (after the header)
- Use Flexbox to layout the elements on the page in both sections
- Stick to the Figma design as much as possible
- Implement interactions to buttons (enable, hover, etc.)

### General requirements
- Make sure that there are no linter errors
- Use correct GitHub Flow
- Create a descriptive README file
- Document my work in a professional way
- Open a pull request and ask for a review

### CODE REVIEW

File: style.css [hunk 1]
```diff
@@ -0,0 +1,139 @@
...
+  cursor: pointer;
+}
+
+/* Introduction */
+#intro {
+  background-color: #1c1a19;
+  background-image: url("images/backgrounds/bg1_mobile.svg");
+  background-repeat: no-repeat;
+  background-size: 26%;
+  background-position: 100% 115%;
```
Comment (keroloussamy) [inline style.css:70]: Kindly you can make the `background-size: 26%;` role a little bit bigger, like `background-size: 35%;`, to make your design like the Figma design.

File: main.html [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+                </li>
+                <li class="link">
+                    <a href="https://twitter.com/jordan12AC"><img src="images/Icons/twitter_icon.svg" alt="twitter icon"></a>
+                </li>
+                <li class="link">
+                    <a href="https://medium.com/"><img src="images/Icons/medium_icon.svg" alt="medium_icon"></a>
+                </li>
+            </ul>
+        </div>
+    </section>
```
Comment (keroloussamy) [inline main.html:57]: Kindly separate between the two sections, make the first one with the `header` tag, and the second one with the `section` tag.
Like 
```
<header>
  <nav>.....</nav>
</header>
<section>
 ......
</section>
```

File: main.html [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+                <li class="time">9:41</li>
+                <li class="fas-container">
+                    <i class="fas fa-signal"></i>
+                    <i class="fas fa-wifi"></i>
+                    <i class="fas fa-battery-full"></i>
+                </li>
+            </ul>
+            <header id="tool-bar">
+                <a class="logo" href="#">John Doe</a>
+                <i class="fas fa-bars"></i>
```
Comment (keroloussamy) [inline main.html:27]: Kindly make the hamburger button `<i class="fas fa-bars"></i>`, inside `nav` tag.
Like  `<nav><i class="fas fa-bars"></i></nav>`.

**Review by keroloussamy — CHANGES_REQUESTED** [review]
Hi @Jord4N-AC, 

Good job so far, you did great work 💯!
There are some issues that you still need to work on to go to the next project but you are almost there!

## To highlight 🎯

- No linter errors.✔️
- Good code design and implementation. ✔️

## Required Changes ♻️
_Check the comments under the review._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: main.html [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+            <li class="fas-container">
+                <i class="fas fa-signal"></i>
+                <i class="fas fa-wifi"></i>
+                <i class="fas fa-battery-full"></i>
+            </li>
+        </ul>
+        <div id="tool-bar">
+            <a class="logo" href="#">John Doe</a>
+            <nav><i class="fas fa-bars"></i></nav>
+        </div>
```
Comment (gemmen29) [inline main.html:27]: [OPTIONAL] Kindly wrap this part with the `nav` tag for semantic purposes

**Review by gemmen29 — APPROVED** [review]
Hi @Jord4N-AC, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Optional suggestions
_Please check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**